pennylane/ops/qubit/matrix_ops.py

```diff
@@ -474,8 +491,10 @@ def compute_matrix(*params, **hyperparams):
             )
             col2 = qml.math.vstack(
                 [
-                    qml.math.sqrt_matrix(qml.math.eye(d1, like=A) - A @ qml.math.conj(A).T),
-                    -qml.math.conj(A).T,
+                    qml.math.sqrt_matrix(
+                        qml.math.eye(d1, like=A) - A @ qml.math.transpose(qml.math.conj(A))
```

Comment:
It might be easier to assign `qml.math.transpose(qml.math.conj(A))` to a variable `Adg` and then use it. This leads to cleaner code and saves the computation for finding conjugate-transpose repeatedly.